expat/configure.ac

```diff
@@ -42,89 +43,96 @@ dnl If the API has changed, increment LIBCURRENT and set LIBREVISION to 0
 dnl
 dnl If the API changes compatibly (i.e. simply adding a new function
 dnl without changing or removing earlier interfaces), then increment LIBAGE.
-dnl 
+dnl
 dnl If the API changes incompatibly set LIBAGE back to 0
 dnl
 
 LIBCURRENT=7   # sync
 LIBREVISION=8  # with
 LIBAGE=6       # CMakeLists.txt!
 
-CPPFLAGS="${CPPFLAGS} -DHAVE_EXPAT_CONFIG_H"
-AC_CONFIG_HEADER(expat_config.h)
+AX_APPEND_FLAG([-DHAVE_EXPAT_CONFIG_H], [CPPFLAGS])
+AC_CONFIG_HEADER([expat_config.h])
 
-sinclude(conftools/ac_c_bigendian_cross.m4)
+AM_PROG_AR
+AC_PROG_INSTALL
+AC_PROG_LN_S
+AC_PROG_MAKE_SET
+AC_PROG_MKDIR_P
```

Comment:
If the project sticks with multiple makefiles, it should put a line:

```
@SET_MAKE@
```

in each `Makefile.in` that makes use of `$(MAKE)`; Automake performs this addition automatically.  `AC_PROG_SET_MAKE` defines such substitution.  Usually, I just call this macro in every `configure.ac` I write and never think of it again.

`AC_PROG_MKDIR_P` is currently unused; I just always include it...  I will drop it, but it could be useful if we switch to a single makefile.

Reply:
I wonder if `AC_PROG_MAKE_SET` could make things worse for users of macOS with command `gmake` but not command `make` (that I think is common). Will this remaing working for them?:
```
./configure
gmake all check
gmake install
```
I hope the `if Make does not do so automatically` in `Define SET_MAKE to set ${MAKE} if Make does not do so automatically` is key here and that gmake does define `MAKE` on macOS.